tests/foreman/virtwho/cli/test_esx_sca.py

```diff
@@ -502,3 +505,93 @@ def test_positive_remove_env_option(
         env_warning = f"Ignoring unknown configuration option \"{option}\""
         result = target_sat.execute(f'grep "{env_warning}" /var/log/messages')
         assert result.status == 1
+
+    @pytest.mark.tier2
+    def test_positive_rhsm_username_option(
+        self, module_sca_manifest_org, form_data_cli, target_sat
+    ):
+        """Verify rhsm_username options in the configure file"
+
+        :id: 2d0d3126-859f-4092-9196-a553bc1d3bd9
+
+        :expectedresults:
+            1. virt-who config belong to a same org with the same rhsm_username
+            2. virt-who config belong to a different org with the different rhsm_username
+            3. hypervisors and guest mapping info send by group of org and service account exist in rhsm.log
+            4. verify delete virt-who config and remain only one virt-who config, service account virt_who_reporter_X exist
+            5. verify delete all virt-who config, service account virt_who_reporter_X has be deleted
```

Comment:
```suggestion
            1. virt-who config belongs to the same org with the same rhsm_username
            2. virt-who config belongs to a different org with the different rhsm_username
            3. rhsm.log contains the correct orgs where hypervisors-guest mapping is sent to
            4. rhsm.log contains the correct account
            5. After deleting a virt-who config, verify that only one rhsm user is left
```

Reply:
@lhellebr I have updated the step 5 and accept the commit suggestion